package.json

```diff
@@ -11,8 +11,9 @@
   },
   "bin": {
     "git-mob": "bin/mob.js",
+    "git-mob-print": "bin/mob-print.js",
     "git-solo": "bin/solo.js",
-    "git-mob-print": "bin/mob-print.js"
+    "git-add-author": "bin/add-author.js"
```

Comment:
Since the command will be directly after `git` it will read `git add-author` which might be misleading to changing the git author in someway so perhaps we can change it to `git add-coauthor` or `git mob --add-coauthor`.